core/common/src/main/java/alluxio/underfs/MultiRangeObjectInputStream.java

```diff
@@ -120,10 +131,10 @@ protected abstract InputStream createStream(long startPos, long endPos)
   /**
    * Block size for reading an object in chunks.
    *
-   * @return block size in bytes
+   * @return chunk size in bytes
    */
-  private long getBlockSize() {
-    return mBlockSize;
+  private long getChunkSize() {
```

Comment:
Is this used anywhere?

Reply:
We can get rid of this now. I think before the removal of singleton configuration it was used as a convenience method to just get the number of bytes from the configuration.